lib/utils/initialAppConfig.ts

```diff
@@ -15,9 +15,35 @@
 */
 
 import { MLSpaceConfig } from './configTypes';
+import * as fs from 'fs';
 
 
 export function generateAppConfig (mlspaceConfig: MLSpaceConfig) {
+    //Check for properties set in config.json and default to that value if it exists
+    let clusterConfig = undefined;
+    let applicationList = [];
+    if (fs.existsSync('lib/resources/config/cluster-config.json')) {
+        clusterConfig = JSON.parse(
+            fs.readFileSync('lib/resources/config/cluster-config.json').toString('utf8')
+        );
+        if (clusterConfig['applications']) {
+            for (const application of clusterConfig['applications']) {
+                applicationList.push({'M': {'name': {'S': application['Name']}}});
+            }
+        }
+    }
+    // If the cluster config doesn't exist, default to this list
+    if (!clusterConfig) {
```

Comment:
This can probably just be in the `else` of the above conditional correct? If the above is true then `clusterConfig` is set.

Reply:
ah yeah good point will do that